core/src/main/java/com/linecorp/armeria/internal/annotation/DefaultExceptionHandler.java

```diff
@@ -58,11 +61,15 @@ public HttpResponse handleException(ServiceRequestContext ctx, HttpRequest req,
             return ((HttpResponseException) cause).httpResponse();
         }
 
+        log(log -> log.warn("{} Unhandled exception from an annotated service:", ctx, cause));
+
+        return HttpResponse.of(HttpStatus.INTERNAL_SERVER_ERROR);
+    }
+
+    private static void log(Consumer<Logger> logConsumer) {
```

Comment:
Thanks for your reply @trustin. I initially did the same and then realized that we use `{}` to format string with var-args parameters of `Logger#warn(String message, Object... varargs)`. In this particular case we have two log statements in this class and we are only using `ctx` in var-arg but though it would be nice to be more generic. Please let me know if you still think this is too much. Will be happy re revert this particular change back.

Reply:
`org.slf4j.Logger` has `warn(String format, Object arg1, Object arg2)`. Anyways, I'm OK with your suggestion.

Reply:
Either way is fine for me. 😄